source/common/world/Dimension.hpp

```diff
@@ -55,8 +57,8 @@ class Dimension : public gk::ISerializable {
 		const std::string &sky() const { return m_sky; }
 		void setSky(const std::string &sky) { m_sky = sky; }
 
-		float gravity() const { return m_gravity; }
-		void setGravity(float gravity) { m_gravity = gravity; }
+		DimensionPhysics physics() const { return m_physics; }
+		void setPhysics(DimensionPhysics physics) { m_physics = physics; }
```

Comment:
`const DimensionPhysics &` in the return type for the getter, and in the parameter for the setter